business/checkers/authorization/namespace_method_checker.go

```diff
@@ -0,0 +1,140 @@
+package authorization
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"strings"
+
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+)
+
+type NamespaceMethodChecker struct {
+	AuthorizationPolicy kubernetes.IstioObject
+	Namespaces          models.NamespaceNames
+}
+
+func (ap NamespaceMethodChecker) Check() ([]*models.IstioCheck, bool) {
+	checks, valid := make([]*models.IstioCheck, 0), true
+
+	// Getting rules array. If not present, quitting validation.
+	rulesStct, ok := ap.AuthorizationPolicy.GetSpec()["rules"]
+	if !ok {
+		return checks, valid
+	}
+
+	// Getting slice of Rules. Quitting if not an slice.
+	rules := reflect.ValueOf(rulesStct)
+	if rules.Kind() != reflect.Slice {
+		return checks, valid
+	}
+
+	for ruleIdx := 0; ruleIdx < rules.Len(); ruleIdx++ {
+		rule, ok := rules.Index(ruleIdx).Interface().(map[string]interface{})
+		if !ok || rule == nil {
+			continue
+		}
+
+		if rule["from"] != nil {
+			fromChecks, fromValid := ap.validateFromField(ruleIdx, rule["from"])
+			checks = append(checks, fromChecks...)
+			valid = valid && fromValid
+		}
+
+		if rule["to"] != nil {
+			fromChecks, fromValid := ap.validateToField(ruleIdx, rule["to"])
```

Comment:
Minor, but on this condition, I'd expect a "toChecks", "toValid", if you want to maintain that way of naming.